bot/controllers/welcome.js

```diff
@@ -0,0 +1,33 @@
+const BaseController = require('../baseController.js');
+const Command = require('../baseCommand.js');
+
+
+class WelcomeController extends BaseController {
+  constructor(message) {
+    // Call BaseController constructor
+    super(message);
+
+    // Aliasing 'this' as controller to allow for binding in actions
+    const controller = this;
+
+    // Array of all commands, see baseCommand.js for prototype
+    this.commands = [
+      new Command(
+        '!welcome',
+        '!welcome',
+        'Welcome Message',
+        'Resend the welcome message you recieved when you joined the server.',
+        this.welcomeAction.bind(controller),
+        'dm',
+      ),
+    ];
+  }
+
+  // this message will be sent to the user's dm with their username
+  welcomeAction() {
+    const { message } = this;
+    return `Welcome to the server, ${message.author.username}`;
```

Comment:
This message differs from the actual welcome message, so this is not actually a resending of that message. ITs these small deviation, combined with the help message that you are defining this as how you 'resend' that message that will confuse users.